src/components/IdeaCardComponent.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, CardActions } from '@mui/material';
+import ShowMoreButton from './Buttons/ShowMore';
+
+function IdeaCard(props) {
+    const { idea } = props;
+
+    // Null checks
+    if (!idea) return null;
+
+    const description = idea.description.slice(0, 150); // Limit length of the description here
+    const altText = `picture of ${idea.header}`;
+    const URL = 'https://fund-raising-project-server-side.azurewebsites.net/';
+    const href = `${URL}ideas/${idea.id}`;
```

Comment:
Can you use relative path here, instead of an absolute path?

Reply:
Fixed.